src/pages/home/report/ReportActionCompose.js

```diff
@@ -673,8 +679,8 @@ class ReportActionCompose extends React.Component {
                         </Tooltip>
                     </View>
                 </View>
-                <View style={[styles.chatItemComposeSecondaryRow, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter]}>
-                    <OfflineIndicator />
+                <View style={[styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter]}>
+                    {!this.props.isSmallScreenWidth && <OfflineIndicator containerStyles={[styles.chatItemComposeSecondaryRow]} />}
```

Comment:
Also, why this unproposed changes?

Reply:
I copied/pasted `ReportActionCompose.js` from the upstream and it has changed [here](https://github.com/Expensify/App/blob/ba5a38f4a8f8eca2fdc63eb2b45aaccc85fe7997/src/pages/home/report/ReportActionCompose.js#L680-L681)